contracts/omen/OmenStakableTokenValidator.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
I have a doubt here. The stakable token in this case should be the Omen AMM's LP token, why are we validating it by looking it up in the token registry? Do we want to register and verify every single LP token for every single LP there? I thought the idea would be to verify that the stakable token was indeed a valid Omen AMM LP token.

Reply:
Yeah, I'd like to verify that it's an LP token, but not sure of a reliable way to do so. I assumed there may be some security benefit to still use the validator, but not sure. Would there be reason against not using a stakable token validator at all for Omen?